web3/providers/base.py

```diff
@@ -72,9 +97,9 @@ def encode_rpc_request(self, method, params):
         encoded = FriendlyJsonSerde().json_encode(rpc_dict)
         return to_bytes(text=encoded)
 
-    def isConnected(self):
+    def isConnected(self) -> bool:
         try:
-            response = self.make_request('web3_clientVersion', [])
+            response = self.make_request(RPCEndpoint('web3_clientVersion'), [])
```

Comment:
Probably nice to do something like have a module that has all of these pre-wrapped for re-use.